cmp/models.py

```diff
@@ -103,18 +102,47 @@ def __str__(self):
 
 
 class SoldierDeath(models.Model):
-    soldier = models.OneToOneField(Soldier, on_delete=models.CASCADE, primary_key=True)
-    date = models.DateField(null=True, blank=True)
-    company = models.ForeignKey(Company, blank=True, null=True, default="UNKNOWN", on_delete=models.CASCADE) 
-    cemetery = models.ForeignKey(Cemetery, blank=True, null=True, default=110, on_delete=models.CASCADE)
+    soldier  = models.OneToOneField(Soldier, on_delete=models.CASCADE, related_name='soldierdeath')
+    date     = models.DateField(null=True, blank=True)
+    company  = models.ForeignKey(Company, blank=True, null=True, default="UNKNOWN", on_delete=models.CASCADE, related_name='companies')
```

Comment:
The way you are using related_name here is probably not what you would like to happen.
from a company object, you probably want to query company.soldierdeaths.all() to get all related soldierdeaths. But based on this code you will need to query company.companies to get all related soldierdeaths.

better to remove all related names then in Meta: add a default related name like so:
```
class Meta:
    default_related_name = 'soldierdeaths'
```